README.md

```diff
@@ -24,6 +24,36 @@ The bootstrapper also includes updated Scrolls library that allows posting to em
 
 See the example in the new project created by the bootstrapper for more information. 
 
+#### Modules
+
+There are currently 4 modules in the project:
+
+- app - Main Android application and UI
+- app-common - Code shared between different modules
+- app-infrastructure - Communications with the outside world. Data storage, network, etc
+- app-domain - Core application classes and logic
```

Comment:
Maybe something like
```suggestion
- app-domain - Application domain and business logic. Use-cases, entities, gateways, etc.
```
would be clearer?

Reply:
I'd like to avoid the works business logic, etc. Doesn't really matter what "logic" is written there, but "business" has a concrete meaning which might take the attention away from the actual value of creating reusable logic blocks. Doesn't matter if it is regarded as app's internal logic or "business" logic.

"business" logic usually excludes data handling, etc. But depending on the app's scale, this core (for a lack of a better word :D) logic can actually include caching decisions, too